iml-wasm-components/iml-grafana-chart/src/lib.rs

```diff
@@ -12,13 +12,14 @@ pub static GRAFANA_DASHBOARD_NAME: &str = "stratagem";
 pub fn grafana_chart<T>(
     dashboard_id: &str,
     dashboard_name: &str,
+    fs_name: &str,
     refresh: &str,
     panel_id: u16,
     width: &str,
     height: &str,
 ) -> Node<T> {
     iframe![attrs! {
-       At::Src => format!("{}d-solo/{}/{}?orgId=1&refresh={}&panelId={}", grafana_root(), dashboard_id, dashboard_name, refresh, panel_id),
+       At::Src => format!("{}d-solo/{}/{}?orgId=1&var-fs_name={}&refresh={}&panelId={}", grafana_root(), dashboard_id, dashboard_name, fs_name, refresh, panel_id),
```

Comment:
After this change, the `grafana_chart fn` will have a var specific to filesystems. Is there a way we can pass extra vars?

What if instead of passing the qs params individually you pass in something that implements `Serialize` and use https://docs.rs/serde_urlencoded/0.6.1/serde_urlencoded/fn.to_string.html

Reply:
I like the idea. I'll update.